lib/utils/app_string_res.dart

```diff
@@ -27,6 +27,7 @@ const lastnameerror='Please enter a valid last name';
 
 const emailerror='Please enter a valid email';
 const address='Enter address manually';
+const addresserror='Please enter a valid address';
```

Comment:
make all string constant readable : 
like 
`const pls_enter_valid_address ='Please enter a valid address';`

Reply:
try to make variable value more readable

Reply:
ok deepak ji